index.js

```diff
@@ -112,6 +107,13 @@
     };
   }
 
+  //  clean :: Object -> Object
+  function clean(x) {
+    var k, o = Object.create(null);
+    for (k in x) o[k] = x[k];
```

Comment:
In my draft, we had the `has` check here, but it turns out we don't need it, as we're passing objects where no properties in the prototype chain are enumerable. I'm not sure if I should add it back as a guard against mutated prototypes, as has been good practice for a long time.